src/Sql/Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -102,9 +103,8 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseCr
         /// Gets or sets the read scale option to assign to the Azure SQL Database
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The read scale option to assign to the Azure SQL Database.(Enabled/Disabled)")]
-        [ValidateNotNullOrEmpty]
-        public DatabaseReadScale ReadScale { get; set; }
+            HelpMessage = "If enabled, connections that have application intent set to readonly in their connection string may be routed to a readonly secondary replica. This property is only settable for Premium and Business Critical databases.")]
+        public DatabaseReadScale? ReadScale { get; set; }
```

Comment:
Don't do this - this is being flagged as a breaking change, and there is no need to do this at all.  Use this pattern to determine whether a parameter was provided by the user:  https://github.com/Azure/azure-powershell/blob/e8ff3ad1e975135a616ec6a52bd4e0056bfc3892/src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs#L157

Reply:
This has been addressed.  Please have a look.